src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }

```

Comment:
In this case it's better to remove these setters and swap them for a constructor with all fields.